### ISSUE TITLE
DeprecationWarning when khoros.errors.exceptions imported in pytest

### ISSUE THREAD

**Issue by rzuckerm** [jeffshurtliff/khoros#57]
## Related Function(s) or Method(s)
- `khoros.errors.exceptions`

## Details
I see the following warning in pytest when I import `khoros.error.exceptions`:

> .../lib/python3.10/site-packages/khoros/objects/base.py:79: DeprecationWarning: The 'khoros.objects.base.Mapping' class has been deprecated. Use 'khoros.structures.nodes.Mapping' instead.
    warnings.warn("The 'khoros.objects.base.Mapping' class has been deprecated. Use " +

## Steps to Reproduce
I was able to reproduce this with the following:
```python
from khoros.errors.exceptions import KhorosError


def test_nothing():
    pass
```

I ran this:
```
pytest -vvl test_nothing.py
```

## Expected Behavior
No deprecation warning should be displayed.

## Suggestions or Proposals
N/A

**Comment by jeffshurtliff** [comment]
This has been fixed in **v5.0.0rc1**, as shown below.

``` python
(venv) shurtj@Jeffs-MacBook-Air ~/D/r/a/k/tests> pytest -vvl test_nothing.py                                                               2966ms  Tue Sep 20 14:31:56 2022
=========================================================================== test session starts ============================================================================
platform darwin -- Python 3.9.10, pytest-6.2.1, py-1.10.0, pluggy-0.13.1 -- /usr/local/opt/python@3.9/bin/python3.9
cachedir: .pytest_cache
rootdir: /Users/shurtj/Development/rsalink-python/adhoc/khoros/tests
collected 1 item                                                                                                                                                           

test_nothing.py::test_nothing PASSED                                                                                                                                 [100%]

============================================================================ 1 passed in 0.40s =============================================================================
(venv) shurtj@Jeffs-MacBook-Air ~/D/r/a/k/tests> 
```

**Comment by jeffshurtliff** [comment]
This fix was included in **v5.0.0** which is now generally available.

**Comment by rzuckerm** [comment]
Thanks, @jeffshurtliff , I'm currently using this new release and it definitely fixes my issue.
